lib/saml/user.rb

```diff
@@ -21,7 +21,7 @@ class User
       'dslogon_loa3' => { loa_current: '3', sign_in: { service_name: 'dslogon' } },
       'myhealthevet' => { loa_current: nil, sign_in: { service_name: 'myhealthevet' } },
       # SSOe context
-      'urn:oasis:names:tc:SAML:2.0:ac:classes:Password' => { loa_current: nil, sign_in: { service_name: 'ssoe' } },
+      'urn:oasis:names:tc:SAML:2.0:ac:classes:Password' => { loa_current: '1', sign_in: { service_name: 'ssoe' } },
```

Comment:
This seems like a pretty significant change, but i think this whole section (SSOe) is new so guessing the SAML responses are in flux?

Reply:
The SAML responses are relatively set, but we are debugging and making adjustments throughout the testing process to get unblocked. In particular, they currently do not pass a custom AuthnContext, and only pass `urn:oasis:names:tc:SAML:2.0:ac:classes:Password`.  There are active development plans on their end to pass through the custom AuthNContext.